docs/deployment/openshift.md

```diff
@@ -87,10 +87,6 @@ worker:
   image: your/invenio-image
 ```
 
-!!! error "`INVENIO_SITE_HOSTNAME` needs to be set up"
-    The Helm charts need to be updated to support this variable in the `values.yaml`. In the meantime,
-    this value can be set manually in the ConfigMap, or hardcode it in the web charts. The value should
-    be the URL or your instance (Same as the `host` above).
```

Comment:
Major: What should we do here? It's declared [here](https://github.com/inveniosoftware/helm-invenio/blob/master/invenio/templates/configuration.yaml#L127) in invenio-helm, but seems not used anywhere else. Could someone more expert confirm if we can delete it?

Reply:
I can't comment on this, but @ppanero is the man for the occasion.